examples/vcl.py

```diff
@@ -173,14 +185,15 @@ def callback(_i, _ii, e):
         print("\t" + "\t".join([f"{100 * e:.2f}%" for e in test_errors[i-1, :i]]))
 
         if inference == "mean-field":
-            bnn.update_prior(tyxe.priors.DictPrior(bnn.net_guide.get_detached_distributions(
-                tyxe.util.pyro_sample_sites(bnn.net))))
+            site_names = tyxe.util.pyro_sample_sites(bnn)
+            bnn.update_prior(tyxe.priors.DictPrior(bnn.net_guide.get_detached_distributions(site_names)))
 
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
     parser.add_argument("--root", default=ROOT)
     parser.add_argument("--dataset", choices=["mnist", "cifar"], required=True)
     parser.add_argument("--inference", choices=["mean-field", "ml"], required=True)
+    parser.add_argument("--test", action='store_true')
```

Comment:
I think I'd rather add an explicit `num-epochs` argument that is by default `60` for CIFAR and `600` for MNIST since that is generally useful and more explicit as to what it does based on the name.